.eslintrc

```diff
@@ -94,3 +94,4 @@ rules:
   "no-path-concat": 0
   "no-process-exit": 0
   "eol-last": 2
+  "key-spacing": 0
```

Comment:
Ah neat, I hadn't seen [key-spacing](https://github.com/eslint/eslint/blob/master/docs/rules/key-spacing.md) yet!
Any reason you'd want to disable that rule instead of using the default value of a single space after colons?